src/__swaps__/screens/Swap/hooks/useEstimatedGasFee.ts

```diff
@@ -55,11 +55,8 @@ export function useSwapEstimatedGasFee(overrideGasSettings?: GasSettings) {
   const { assetToSell, quote, chainId = preferredNetwork || ChainId.mainnet } = useSyncedSwapQuoteStore();
   const gasSettings = useSelectedGas(chainId);
 
-  const { data: estimatedGasLimit, isFetching } = useSwapEstimatedGasLimit({ chainId, assetToSell, quote });
+  const estimatedGasLimit = useSwapEstimatedGasLimit({ chainId, assetToSell, quote });
```

Comment:
I know you added `notifyOnChangeProps` to prevent this in places, but any reason to verbosely define these? IIIRC we discussed how tanstack automatically handles this under the hood depending on wht props you destructure. https://tanstack.com/query/latest/docs/framework/react/guides/migrating-to-react-query-4#notifyonchangeprops-property-no-longer-accepts-tracked-as-a-value

Reply:
So if you don't care to listen for `isFetching`, you can just grab the `data` object.

```
  const { data: estimatedGasLimit } = useSwapEstimatedGasLimit({ chainId, assetToSell, quote });
```

Reply:
Reason I'm asking is because I remember trying to add the loading state based on `isFetching` and it wasn't working because it wasn't detecting the change, and since we weren't passing a `notifyOnChangeProps` prop here and I had to go into the hook itself to see it was hardcoded to `data` (which is still is). I think this imposes complicated DX.